lib/vfs/src/lib.rs

```diff
@@ -1,42 +1,54 @@
 use std::any::Any;
 use std::ffi::OsString;
 use std::fmt;
-use std::io::{self, Read, Seek, Write};
+use std::io;
 use std::path::{Path, PathBuf};
+use std::pin::Pin;
+use std::task::Context;
+use std::task::Poll;
 use thiserror::Error;
 
-#[cfg(all(not(feature = "host-fs"), not(feature = "mem-fs")))]
-compile_error!("At least the `host-fs` or the `mem-fs` feature must be enabled. Please, pick one.");
-
-//#[cfg(all(feature = "mem-fs", feature = "enable-serde"))]
-//compile_warn!("`mem-fs` does not support `enable-serde` for the moment.");
-
+pub mod arc_file;
+pub mod arc_fs;
+pub mod builder;
+pub mod empty_fs;
 #[cfg(feature = "host-fs")]
 pub mod host_fs;
-#[cfg(feature = "mem-fs")]
```

Comment:
so "mem-fs" is always present? That's a good idea.

Reply:
Yep, doesn't require any extra dependencies, so it makes sense to have it always available.